recipes/llnl-sina/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "llnl-sina" %}
+{% set version = "1.9.1" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/LLNL/Sina/archive/{{ version }}.tar.gz
+    sha256: 28ca48c89b8e8280b41458f8a8eabf51408c68bc3a45a433258eb548b5447e8e
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - cd python
+    - {{ PYTHON }} -m pip install . -vv
+ 
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - six
+    - sqlalchemy
+    - enum34
```

Comment:
We no longer build any Python <3.4 so I guess it is safe to remove this.

Reply:
@ocefpaf internally we still need to build for Python<3.4 do you mind if we keep this one in? I would like to use the same recipe everywhere. Thanks!

Reply:
Sure. That is not a problem.